buildenv/jenkins/README.md

```diff
@@ -0,0 +1,194 @@
+    - `Jenkins compile`
+    - `Jenkins test sanity`
+    - `Jenkins test extended`
+- Request a single platform instead of "all"
+    - `Jenkins test sanity zlinux`
+    - `Jenkins test sanity plinux`
+- You can also request a "Compile & Sanity" build from an openj9-openjdk-jdk9 PR or an openj9-omr PR.
+
+##### Dependent Changes
+
+- If you have dependent change(s) in either eclipse/omr, eclipse/openj9-omr, or ibmruntimes/openj9-openjdk-jdk9, you can build & test with all needed changes
+- Request a build by including the PR ref in your trigger comment
+- Ex. Dependent change in OMR Pull Request `#123`
+    - `Jenkins test sanity depends eclipse/omr#123`
+- Ex. Dependent change in OpenJ9-OMR Pull Request `#456`
+    - `Jenkins test sanity depends eclipse/openj9-omr#456`
+- Ex. Dependent change in OpenJDK Pull Request `#789`
+    - `Jenkins test sanity depends ibmruntimes/openj9-openjdk-jdk9#789`
+- Ex. Dependent changes in OMR and OpenJDK
+    - `Jenkins test sanity depends eclipse/omr#123 ibmruntimes/openj9-openjdk-jdk9#789`
+- Ex. If you have a dependent change and only want one platform, depends comes last
+    - `Jenkins test sanity zlinux depends eclipse/omr#123`
+
+**Note:** Dependent changes can only be requested from an OpenJ9 Pull Request
+
+### Overview of Builds
+
+#### Build
+
+- Build-linux_390-64_cmprssptrs
+    - [![Build Status](https://ci.eclipse.org/openj9/buildStatus/icon?job=Build-linux_390-64_cmprssptrs)](https://ci.eclipse.org/openj9/job/Build-linux_390-64_cmprssptrs)
+    - Description:
+        - Compiles on linux_390-64_cmprssptrs
+        - Archives the SDK and test material
+    - Trigger:
+        - This job is used in other pipelines but can be launched manually
+
+- Build-linux_ppc-64_cmprssptrs_le
+    - [![Build Status](https://ci.eclipse.org/openj9/buildStatus/icon?job=Build-linux_ppc-64_cmprssptrs_le)](https://ci.eclipse.org/openj9/job/Build-linux_ppc-64_cmprssptrs_le)
+    - Description
+        - Compiles on linux_ppc-64_cmprssptrs_le
+        - Archives the SDK and test material
+    - Trigger:
+        - This job is used in other pipelines but can be launched manually
+
+#### Infrastructure
+
+- Mirror-OMR-to-OpenJ9-OMR
+    - [![Build Status](https://ci.eclipse.org/openj9/buildStatus/icon?job=Mirror-OMR-to-OpenJ9-OMR)](https://ci.eclipse.org/openj9/job/Mirror-OMR-to-OpenJ9-OMR)
+    - Description:
+        - Mirrors eclipse/omr/master to eclipse/openj9-omr/master
+        - Triggers `Pipeline-OMR-Acceptance` when there is new content        
+    - Trigger:
+        - Build periodically, 15 minutes
+
+- Mirror-OpenJ9-Website-to-Eclipse
+    - [![Build Status](https://ci.eclipse.org/openj9/buildStatus/icon?job=Mirror-OpenJ9-Website-to-Eclipse)](https://ci.eclipse.org/openj9/job/Mirror-OpenJ9-Website-to-Eclipse)
+    - Description:
+        - Mirrors github.com/eclipse/openj9-website to the Eclipse.org repo
+    - Trigger:
+        - Poll Github repo for changes
+
+- Promote-OpenJ9-OMR-master-to-openj9
```

Comment:
Can we reorganize this a little so the two OMR related builds are together and the website one comes after?

Also, can this describe the tag laid down on merges?